pkg/cmd/pr/merge/merge.go

```diff
@@ -253,46 +269,60 @@ func (m *mergeContext) merge() error {
 		setCommitBody: m.opts.BodySet,
 	}
 
-	// get user input if not already given
-	if m.opts.InteractiveMode {
-		apiClient := api.NewClientFromHTTP(m.httpClient)
-		r, err := api.GitHubRepo(apiClient, m.baseRepo)
-		if err != nil {
-			return err
-		}
-
-		payload.method, err = mergeMethodSurvey(r)
-		if err != nil {
-			return err
-		}
+	if m.addToMergeQueue() && !m.opts.InteractiveMode {
+		_ = m.warnf("%s a merge strategy cannot be set when adding a pull request to the merge queue\n", m.cs.FailureIconWithColor(m.cs.Red))
+		return cmdutil.SilentError
+	}
 
-		m.deleteBranch, err = deleteBranchSurvey(m.opts, m.crossRepoPR, m.localBranchExists)
-		if err != nil {
-			return err
-		}
+	var err error
```

Comment:
because I removed `r, err := api.GitHubRepo(apiClient, m.baseRepo)` the assignment of `payload.method, err := mergeMethodSurvey(m.baseRepo)` fails because it is assigning to a field on a struct